Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 1
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	all
	1	report
	5	run_bt
	7

rule run_bt:
    input: data/t1_N400_clade0.2_b1.5_d0.5_rate10.trait, config_files/pr1.bt
    output: t1_N400_clade0.2_b1.5_d0.5_rate10_pr1.bt, out/bt/t1_N400_clade0.2_b1.5_d0.5_rate10_pr1.summary.txt
    log: log/t1_N400_clade0.2_b1.5_d0.5_rate10_pr1.bt.log
    jobid: 6
    benchmark: bench/t1_N400_clade0.2_b1.5_d0.5_rate10_pr1.bt.txt
    wildcards: sample=t1_N400_clade0.2_b1.5_d0.5_rate10, n=1

Terminating processes on user request.
Error in rule run_bt:
    jobid: 6
    output: t1_N400_clade0.2_b1.5_d0.5_rate10_pr1.bt, out/bt/t1_N400_clade0.2_b1.5_d0.5_rate10_pr1.summary.txt
    log: log/t1_N400_clade0.2_b1.5_d0.5_rate10_pr1.bt.log

RuleException:
CalledProcessError in line 209 of /Users/jeremias/Dropbox/Bioinformatics/Reports/bm_sim/Snakefile:
Command ' set -euo pipefail;  /Users/jeremias/.pyenv/versions/anaconda3-5.0.0/bin/python /Users/jeremias/Dropbox/Bioinformatics/Reports/bm_sim/scripts/.snakemake.pud5yyeb.run_bt.py ' returned non-zero exit status 1.
  File "/Users/jeremias/Dropbox/Bioinformatics/Reports/bm_sim/Snakefile", line 209, in __rule_run_bt
  File "/Users/jeremias/.pyenv/versions/anaconda3-5.0.0/lib/python3.6/concurrent/futures/thread.py", line 55, in run
Complete log: /Users/jeremias/Dropbox/Bioinformatics/Reports/bm_sim/.snakemake/log/2018-02-05T002401.220109.snakemake.log
